.github/workflows/native-cron-build.yml

```diff
@@ -39,34 +42,21 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
 
+      - name: Run dependency checker
```

Comment:
this should be in https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml?